test/language/testdata/python/ParameterTypes.py

```diff
@@ -0,0 +1,16 @@
+"""Parameter Type."""
+from datetime import datetime
+
+
+ParameterType(
```

Comment:
I'm guessing you got this from [here](https://github.com/cucumber/cucumber-expressions#python).
This is misleading - see https://github.com/cucumber/cucumber-expressions/issues/142 for details.

This file should be removed. We can add it back when/if https://github.com/behave/behave/issues/968 is fixed.